src/App.tsx

```diff
@@ -1,18 +1,30 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
 import { getNumbers } from './utils';
+import { Pagination } from './components/Pagination';
 
-// eslint-disable-next-line @typescript-eslint/no-unused-vars
 const items = getNumbers(1, 42)
   .map(n => `Item ${n}`);
 
 export const App: React.FC = () => {
+  const [currentPage, setCurrentPage] = useState(1);
+  const [perPage, setPerPage] = useState(5);
+
+  const visibleItems
+  = items.filter((item, index) => item
+  && (currentPage - 1) * (perPage) <= index
+  && index <= (currentPage * perPage - 1));
+
+  const onPageChange = (page: number) => {
```

Comment:
```suggestion
  const handlePageChange = (page: number) => {
```
https://javascript.plainenglish.io/handy-naming-conventions-for-event-handler-functions-props-in-react-fc1cbb791364